source/AndroidXProject.cshtml

```diff
@@ -2,7 +2,8 @@
 @using System.Linq
 <Project Sdk="Xamarin.Legacy.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>MonoAndroid9.0;net6.0-android</TargetFrameworks>
+    <TargetFrameworks>MonoAndroid12.0;net6.0-android</TargetFrameworks>
+    <AndroidUseLatestPlatformSdk>True</AndroidUseLatestPlatformSdk>
```

Comment:
Is `<AndroidUseLatestPlatformSdk>` needed?

Apparently it is deprecated and should not be used anymore:
https://github.com/xamarin/xamarin-forms-samples/issues/364